### DISCUSSION TITLE
xpore diffmod module

### DISCUSSION THREAD

**Discussion by Lqh09** [GoekeLab/xpore#77] [Q&A] [answered]
Hi, I set the readcount_min to 3 when using diffmod module. I notice only sites both expressed in two samples can be identified. But I want to find some mod sites that are uniquely expressed in one sample. Is there any way to achieve this? Thanks a lot!

**Comment by jonathangoeke** [comment, marked as answer]
Hi @Lqh09 xPore was designed to study differential modifications. If a position is equally modified, equally unmodified, or not expressed in one of the samples, then xPore will not return these positions. If you are interested in identifying m6A positions in a single sample, then you can look at [m6ANet](https://github.com/GoekeLab/m6anet) which we develop for that scenario (currently available as a pre-release).
Thanks!
Jonathan

**Reply by qhli1** [reply]
Got it, thanks!
